tests/integration_tests/resources/scripts/set_ldap.py

```diff
@@ -0,0 +1,53 @@
+########
+# Copyright (c) 2018 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+import ast
+import time
+import socket
+import argparse
+
+from manager_rest import config
+from manager_rest.flask_utils import setup_flask_app, set_admin_current_user
+
+
+def set_ldap(config_dict):
+    app = setup_flask_app()
+    # mock current user, and reload rest configuration
+    set_admin_current_user(app)
+    config.instance.load_from_file("/opt/manager/cloudify-rest.conf")
```

Comment:
Instead of the string `/opt/manager/cloudify-rest.conf`, maybe use the constant `CONFIG_FILE_LOCATION` that is in `from integration_tests.framework import constants`